src/rest-api/spec/training.spec.ts

```diff
@@ -0,0 +1,83 @@
+import { expect } from 'chai';
+import {} from 'mocha'
+import {} from 'core-js'
+import { Training } from '../../model/training';
+import { Student } from '../../model/student';
+import { Trainer } from '../../model/trainer';
+import { trainingApi } from '../training'
+
+let trainingList : Training[];
+
+beforeEach(() => {
+  trainingList =
+  [
+    {
+      id: 32,
+      name: 'React/Redux',
+      isActive: true,
+      start: new Date(1,1,2017),
+      end: new Date(31,1,2017),
+      students: new Array<Student>(),
+      trainers: new Array<Trainer>()
+    },
+    {
+      id: 12,
+      name: 'Responsive web design',
+      isActive: true,
+      start: new Date(1,2,2017),
+      end: new Date(28,2,2017),
+      students: new Array<Student>(),
+      trainers: new Array<Trainer>()
+    },
+    {
+      id: 33,
+      name: 'AngularJS 2.0',
+      isActive: true,
+      start: new Date(1,3,2017),
+      end: new Date(31,3,2017),
+      students: new Array<Student>(),
+      trainers: new Array<Trainer>()
+    },
+  ];
+});
+
+describe('TrainingApi', () => {
+  it('is defined', () => {
+      expect(trainingApi).not.to.be.undefined;
+  })
+
+  describe('setMockDataSeed', () => {
+    it('set the right mockdata seed', () => {
+      // Act
+      trainingApi.setMockDataSeed(trainingList);
```

Comment:
Is this really necessary, expose a public method to set mock data? What is the reason?